util/cbor.go

```diff
@@ -1,12 +1,11 @@
 package util
 
 import (
-	blockstore "github.com/ipfs/go-ipfs-blockstore"
 	cbor "github.com/ipfs/go-ipld-cbor"
 	mh "github.com/multiformats/go-multihash"
 )
 
-func CborStore(bs blockstore.Blockstore) *cbor.BasicIpldStore {
+func CborStore(bs cbor.IpldBlockstore) *cbor.BasicIpldStore {
```

Comment:
I think you should split out these changes to function signatures (here, and `indigo:mst` and `indigo:repo`) to a separate PR, if they aren't touched by the `indigo:cmd/relay` code.